content/components/forms/buttons.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Buttons"
 asOf: 1.0.0
-description: "Buttons are used to signal actions, submit forms, or trigger new content within the current context."
```

Comment:
Generally speaking, these descriptions should provide a quick overview of what the component is/does. I think it would be fine to maybe instead move this note to the _Implementation notes_ section about the length of button text.